packages/entities/entities-plugins/src/components/free-form/shared/utils.ts

```diff
@@ -1,10 +1,12 @@
+import type { UnionFieldSchema } from 'src/types/plugins/form-schema'
+import { toValue, type MaybeRefOrGetter } from 'vue'
+
```

Comment:
nit(non-blocking): can we unify the code style here by importing the type individually with a `import type` statement instead of `import { type } ` 🤔

Reply:
Sure, most likely it was generated by VSCode, will be fixed in next PR.